paper.md

```diff
@@ -42,69 +42,78 @@ bibliography: paper.bib
 However, given the vastness of chemical space [@ruddigkeit_GDB-17_2012] and its often unsmooth nature (e.g. activity cliffs), it seems unlikely that users will want to be restricted to exclusively operate in an interpolation regime.
 Thus, to encourage adoption of these models, it is crucial to measure performance on more challenging splits as well.
-The general workflow is: (1) Convert each molecule into a vector representation (2) Cluster the molecules based on similarity (3) Train the model on some clusters then evaluate performance on unseen clusters that should be dissimilar to the clusters used for training.
+The general workflow is:
+1. Convert each molecule into a vector representation.
+2. Cluster the molecules based on similarity.
+3. Train the model on some clusters and then evaluate performance on unseen clusters that should be dissimilar to the clusters used for training.
 Although measuring performance on chemically dissimilar compounds/clusters is not a new concept [@meredig2018can; @durdy2022random; @stuyver2022quantum; @tricarico2022construction; @terrones2023low], there are a myriad of choices for the first two steps; our software incorporates many popular representations and similarity metrics to give users freedom to easily explore which combination is suitable for their needs.
 
 # Example Use-Case in Cheminformatics
 
 To demonstrate the difference in performance between interpolation and extrapolation, we apply `astartes` to two relevant cheminformatics datasets and their corresponding tasks.
-For each dataset, a typical interpolative split is generated using a Random sampler and two unique extrapolative splits are generated for comparison. The first uses the cheminformatics-specific Bemis-Murcko scaffold [@bemis1996properties] as calculated by RDKit [@landrum2006rdkit]. <!-- Scaffold splits are a better measure of generalizability compared to random splits [@yang2019analyzing; @wang2020machine; @heid2021machine; @guan2021regio; @artrith2021best; @greenman2022multi]. -->
-The second uses the more general-purpose K-means clustering based on the Euclidean distance of Morgan (ECFP4) fingerprints using 2048 bit hashing and radius of 2 [@morgan1965generation; @rogers2010extended]. For each split, we create 5 different folds (by changing the random seed). The values in Table QM9 and Table RDB7 correspond to the mean $\pm$ one standard deviation calculated across folds.
+For each dataset, a typical interpolative split is generated using random sampling. We also create two extrapolative splits are generated for comparison. The first uses the cheminformatics-specific Bemis-Murcko scaffold [@bemis1996properties] as calculated by RDKit [@landrum2006rdkit]. <!-- Scaffold splits are a better measure of generalizability compared to random splits [@yang2019analyzing; @wang2020machine; @heid2021machine; @guan2021regio; @artrith2021best; @greenman2022multi]. -->
```

Comment:
We also create two extrapolative splits ~~are generated~~ for comparison.

Reply:
Thanks for catching this typo as well!